DNN Platform/Library/Entities/IEventManager.cs

```diff
@@ -167,6 +167,12 @@ public interface IEventManager
         /// <param name="args">The event arguments.</param>
         void OnRoleCreated(RoleEventArgs args);
 
+        /// <summary>
+        /// Fires up when a role was updated.
+        /// </summary>
+        /// <param name="args">The event arguments.</param>
+        void OnRoleUpdated(RoleEventArgs args);

```

Comment:
This will be a breaking change for anyone that's implemented `IEventManager`. I don't expect that it's probably happened outside of tests. However, we need to think about a longer term strategy for adding to these interfaces. Do we need to have much smaller interfaces? Do we need to version interfaces? Do we need to use base classes instead of or in addition to interfaces? 🤔

Reply:
I am not opposed to versioned interfaces myself, it would open up soooo much possibilities. I know it does not look pretty, but that is only for devs, and it would make our lives much easier to revamp some areas of the platform...